includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -914,6 +946,43 @@ public function add_days_to_completion_to_courses_queries( $clauses ) {
 
 		return $clauses;
 	}
+	/**
+	 * Fetch the values required for the total counts added to column headers in lesson reports.
+	 *
+	 * @since  4.2.0
+	 * @access private
+	 *
+	 * @param int $course_id Course Id to filter lessons with.
+	 *
+	 * @return object Row containing the totals for column header.
+	 */
+	public function get_totals_for_lesson_report_column_headers( int $course_id ) {
+		global $wpdb;
+
+		$clean_course_id                = esc_sql( $course_id );
+		$comment_query_piece            = [];
```

Comment:
Why do you split your query here?
Not sure, but looks unnecessary to me as we don't filter these pieces, etc.
I think with a single query string, it would look cleaner.

Reply:
You're right, I'll update the query to be in a single line

Reply:
Moved the query into a wpdb->prepare here https://github.com/Automattic/sensei/pull/4911/commits/87f6148bec2babe54cd0a2c1913e2243cd496eda , kindly take a look. Thanks for your suggestion :)